walkthroughs/week-5-tdd/project/src/test/java/com/google/sps/FindMeetingQueryTest.java

```diff
@@ -34,14 +37,17 @@
   // Some people that we can use in our tests.
   private static final String PERSON_A = "Person A";
   private static final String PERSON_B = "Person B";
+  private static final String PERSON_C = "Person C";
 
   // All dates are the first day of the year 2020.
   private static final int TIME_0800AM = TimeRange.getTimeInMinutes(8, 0);
```

Comment:
can you add inline comments for the raw variables? Example:

TimeRange.getTimeInMinutes(/*argumentName1=*/ 8, /*argumentName2=*/0);

This will help me understand what these numbers mean.

Reply:
done!